code/game/jobs/job_globals.dm

```diff
@@ -76,7 +76,7 @@ GLOBAL_LIST_INIT(supply_positions, list(
 	"Shaft Miner"
 ))
 
-GLOBAL_LIST_INIT(service_positions, (list("Head of Personnel") + (support_positions - supply_positions)))
+GLOBAL_LIST_INIT(service_positions, (support_positions - supply_positions))
```

Comment:
Instead of this being defined on support_positions - supply_positions, support_positions doesn't contain any of the supply_positions any more. Could you delete support_positions and move all its contents to service_positions, and replace all instances of `GLOB.support_positions` with `GLOB.service_positions`?

Reply:
@Contrabang Could you review now, to see if I understood you correctly with these changes?
Tested new changes by:

- Starting up server on full map (no obvious broken stuff)
- **Identification Computer** still looks the same as pictured above
- Syndicate ID card editing still works as expected, but with `Service` instead of `Support` listed as occupation category
- _**WAS NOT ABLE TO TEST**_  jobban panel still working as expected (don't have DB set up). So please review `topic.dm` changes carefully.

Reply:
I'll talk with the other members of dev team and see what we can get set up DB side to move supportdept bans to servicedept bans

Reply:
Thank you. While we’re at it, I don’t think there was a category for Supply positions, unless I misunderstood?
Should consider adding `supplydept` category at the same time?

Reply:
That would be great

Reply:
OK 👍 Will add later tonight, and set up local DB while I’m at it.

Reply:
Okay, it should be fine the way it is, but it would still be appreciated if you could add the supply positions. Basically how it works, is it finds every job in the list, and applies a ban to that job specifically. Basically, there should be no need for a database update

Reply:
@Contrabang Added "Supply" to job list. You are right, tested locally and the database doesn't refer to the depts it seems.